.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -1,14 +1,25 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Propose an idea for a feature or an enhancement
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+**What problem does this address, if any?**
+<!--
+If this feature or enhancement is related to a current problem or pain point,
+please describe it. For example, "I'm always frustrated when ..." or "It is
+currently difficult to ...".
+-->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+**What is the solution?**
+<!--
+Please outline the feature or enhancement that you want and how it addresses any
+problem identified above.
+-->
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+**What are the alternative approaches?**
+<!--
+Please describe the alternative approaches that you have considered. For
+example, is there an existing workaround? Is there a plugin that already does
+this?
+-->
```

Comment:
I think it might be nice to add something along the lines of "Is this useful for 80% of users?". Not sure though. Thoughts?

Reply:
"Who does this affect?"?

Could be a good question right after "What problem does this address ..."

Reply:
I wish we could say "80% of users" but like Dan's approach of "who does this affect" more as a good middle ground.